css/how_to_name_the_classes.md

```diff
@@ -0,0 +1,118 @@
+
+Keep the name as simple as possible and add suffixes to create modifiers of the class
+
+```scss
+// Class base SCSS
+.fjord-btn {...} // stands for: .fjord-button
+```
+
+The general rule is `context-component-modifier`
+
+```scss
+// Class modifiers SCSS
+.fjord-btn-sm {...} // stands for: .fjord-button-small
+.fjord-btn-lg {...} // stands for: .fjord-button-large
+```
+
+If the variations of the class are too many, use a numeric suffix
+
+```scss
+// Class numeric modifiers SCSS
+.fjord-text-1 {...}
+.fjord-text-2 {...}
+.fjord-text-3 {...}
+.fjord-text-4 {...}
+```
+
+### D. Component instances
+
+We always recommend the solution proposed in the previous section, but sometimes it will be necessary to change the standard method
+
+> Imagine we need to create 2 images in 2 different components for the Fjord theme, but the differences between these elements are so uniques that we can't afford to generate the required modifiers because they won't be used anywhere else.
+
+In that case we are going to rely on the Atomic Design concepts to add the desired specifications to the class.
+
+```scss
+// Class instance SCSS
+.fjord-card-image {...}
+.fjord-list-image {...}
+```
+
+### E. Responsive particles
+
+There is a special part of the name is worth mentioning even if we created a dedicated session for this topic: the "responsive particles". [future_link]
+
+The "responsive particle" is a part of the class that can be `sm`, `md`, `lg`, or `xl` (small, medium, large, or extra-large), and it goes between the `class-property-word` and the `class-value-word`
+
+```scss
+// SCSS example
+.col-sm-6 // create a column that occupy 6/12 of the available container width on devices with a screen width of `576px` or above
+.col-md-4 // create a column that occupy 4/12 of the available container width on devices with a screen width of 768px or above
+.mt-3 // add a top margin of 16px on every device
+.mt-md-4 // add a top margin of 24px on devices with a screen width of 768px or above
+.d-lg-none // hide the element on devices with a screen width of 992px or above
+.d-xl-block // show the element on devices with a screen width of 1200px or above
+```
+
+We should create our responsive classes following the same pattern
+
+```scss
+// SCSS example
+.fjord-title-sm-1 {
+	font-size: 20px;
+}
+.fjord-title-md-2 {
+	font-size: 30px;
+}
+// using these 2 classes we can create a title that increase its font size on devices with a screen width of 768px or above
+```
+
+## 3. Good practices
+
+### A. RTL ready
+
+To improve the accessibility of the right-to-left languages we should avoid the words `left` and `right` and use `before`, `after`, `prepend`, `append`, `first`, `last`, `previous`, `next`, `back`, `forward` etc
+
+### B. HTML selectors
+
+Instead of using HTML selectors like `.nav > a` you should prefer defined classes like `.nav-link` the result will be lighter and faster for the browser, it will also limit the styles from _bleeding_ to the children tags
```

Comment:
Needs a full stop after `.nav-link`. "The result" starts a new sentence. Needs an "and" before "it will also" to avoid a run-on sentence.